yarn.lock

```diff
@@ -8609,6 +8616,11 @@ tslib@^1.8.0, tslib@^1.8.1, tslib@^1.9.0:
   resolved "https://registry.yarnpkg.com/tslib/-/tslib-1.13.0.tgz#c881e13cc7015894ed914862d276436fa9a47043"
   integrity sha512-i/6DQjL8Xf3be4K/E6Wgpekn5Qasl1usyw++dAA35Ue5orEn65VIxOA+YvNNl9HV3qv70T7CNwjODHZrLwvd1Q==
 
+tslib@^2.0.1:
```

Comment:
Interesting: https://github.com/microsoft/tslib. Do you happen to know why this gets pulled in by (presumably) `@vue/composition-api`?

Reply:
Yep, [looks](https://github.com/vuejs/composition-api/blob/0796912c37d2d4389331e6e029b8b270b76458c4/package.json#L74) like it is `@vue/composition-api`.